app/views/shared/applications/_listing.html.slim

```diff
@@ -14,9 +14,7 @@
 - content_for :javascripts do
   = javascript_packs_with_chunks_tag 'table_toolbar'
 
-/ FIXME: The first condition does not make any sense but application_plans used to be accessible_plans, now it's accessible_plans.stock
-/ FIXME 2: application_plans returns [] in services/:id/applications even though there are plans
-- show_application_plans = application_plans.size > 0 && !master_on_premises?
+- show_application_plans = application_plans.size > 1 && !master_on_premises?
```

Comment:
I agree that the condition didn't make sense :sweat_smile: 
And I think originally it was `> 1`, according to [this](https://github.com/3scale/system/commit/69aca48b8939dba866cd82490a8eb8f7b1a68166#diff-6c0dc15035a9733326ef465c2b9ed98435040707f08613bc14e8256825e08a5fR21), it was changed [here](https://github.com/3scale/porta/commit/a19b6163ae480269e0dc9ebfd86a8565c095dea3#diff-346a6732164cd2f844a174780398b9fe001f6cf239e07cd7dc21fc9588ba475fR21-R22).

I believe this PR is restoring the originally intended logic.